bluesky/tests/test_callbacks.py

```diff
@@ -151,7 +151,7 @@ def test_table(RE, hw):
         hw.det.precision = 2
         hw.motor.precision = 2
         hw.motor.setpoint.put(0.0)  # Make dtype 'number' not 'integer'.
-        hw.det.put(0.0)  # Make dtype 'number' not 'integer'.
+        hw.det.trigger()  # Make dtype 'number' not 'integer'.
```

Comment:
```suggestion
        hw.det.trigger()
```
It seems the comment no longer applies.

Reply:
The `hw.det.trigger()` call is not necessary with the corresponding ophyd updates. Does it make sense to remove this call, @tacaswell?